ballerina/client.bal

```diff
@@ -0,0 +1,91 @@
+    #
+    # + config - The configurations to be used when initializing the `connector` 
+    # + serviceUrl - URL of the target service 
+    # + return - An error if connector initialization failed 
+    public isolated function init(string serviceUrl, *ConnectionConfig config) returns error? {
+        return self.initiateClient(serviceUrl, config);
+    }
+
+    private isolated function initiateClient(string serviceUrl, *ConnectionConfig config) returns error? = @java:Method {
+        'class: "io.ballerina.lib.milvus.Client"
+    } external;
+
+    # Lists all the collections in the Milvus vector database.
+    #
+    # + return - A list of collection names
+    remote isolated function listCollections() returns string[] = @java:Method {
+        'class: "io.ballerina.lib.milvus.Client"
+    } external;
+
+    // TODO: Add support for both dynamic fields and collection schema when creating a collection
+    # Creates a new collection in the Milvus vector database.
+    #
+    # + request - The request to create a collection
+    # + return - `()` if the collection is created successfully, otherwise an error
+    remote isolated function createCollection(CreateCollectionRequest request) returns error? = @java:Method {
+        'class: "io.ballerina.lib.milvus.Client"
+    } external;
+
+    # Loads a collection in the Milvus vector database.
+    # 
+    # + collectionName - The name of the collection to load
+    # + return - `()` if the collection is loaded successfully, otherwise an error
+    remote isolated function loadCollection(string collectionName) returns error? = @java:Method {
+        'class: "io.ballerina.lib.milvus.Client"
+    } external;
+
+    # Creates an index for a collection in the Milvus vector database.
+    #
+    # + request - The request to create an index
+    # + return - `()` if the index is created successfully, otherwise an error
+    remote isolated function createIndex(CreateIndexRequest request) returns error? = @java:Method {
+        'class: "io.ballerina.lib.milvus.Client"
+    } external;
+
+    # Upserts data into a collection in the Milvus vector database.
+    #
+    # + request - The request to upsert data
+    # + return - `()` if the data is upserted successfully, otherwise an error
+    remote isolated function upsert(UpsertRequest request) returns error? = @java:Method {
+        'class: "io.ballerina.lib.milvus.Client"
+    } external;
+
+    # Deletes data from a collection in the Milvus vector database.
+    # 
+    # + request - The request to delete data
+    # + return - The number of deleted entities if the data is deleted successfully, otherwise an error
+    remote isolated function delete(DeleteRequest request) returns int|error = @java:Method {
+        'class: "io.ballerina.lib.milvus.Client"
+    } external;
+
+    # Searches for data in a collection in the Milvus vector database.
+    # 
+    # + request - The request to search for data
+    # + return - A list of search results
```

Comment:
Returns a array of array ryt?

Reply:
Fixed in [abfa402](https://github.com/ballerina-platform/module-ballerinax-milvus/pull/1/commits/abfa4023b796ed5ccf9a09e858e8a68feb6b9b90)